recipes/prometheus-async/recipe.yaml

```diff
@@ -0,0 +1,48 @@
+context:
+  name: prometheus-async
+  python_min: 3.7
+  version: 22.2.0
+
+package:
+  name: ${{ name|lower }}
+  version: ${{ version }}
+
+source:
+  url: https://pypi.org/packages/source/${{ name[0] }}/${{ name }}/prometheus-async-${{ version }}.tar.gz
+  sha256: b0426370eb3b3bacd99afcf1fcc669c118cb67603cc951a6fe12434e9d4307f2
+
+build:
+  number: 0
+  noarch: python
+  script: ${{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+
+requirements:
+  host:
+    - python ==${{ python_min }}
+    - flit-core >=3.4,<4
+    - pip
+  run:
+    - python >=${{ python_min }}
+    - prometheus_client >=0.8.0
+    - typing_extensions >=3.10.0
+    - wrapt
+
+tests:
+  - python:
```

Comment:
This can be simplified as:

```
tests:
   - python:
       imports:
       - prometheus_async
       pip_check: true
       python_version: ${{ python _min }}.*
```